_data/footer.yml

```diff
@@ -11,6 +11,8 @@
     link: "/team"
   - name: FAQs
     link: "/faqs"
+  - name: Solid Labs
+    link: "/solid-labs"
```

Comment:
Any particular reason why this path includes `solid-`?

See also https://github.com/solid/solidproject.org/issues/249

Reply:
No, the actual page does not have that either, it appears. @Mitzi-Laszlo, did you check whether the link looked as intended after deploying to the staging website?

I'll submit and deploy a fix.